test/edu/ucsb/nceas/metacat/README.md

```diff
@@ -0,0 +1,5 @@
+# Use the `test/edu/ucsb/nceas/metacat` directory only for unit tests and integration tests that do NOT require metacat to be running
+
+For tests that need a running metacat instance, and test networking and replication functionality, use `test/edu/ucsb/nceas/metacatnettest`
+
+NOTE there may currently be tests in the wrong location. Feel free to move them.
```

Comment:
Consider renaming these to: `test/edu/ucsb/nceas/metacat/unit` and `test/edu/ucsb/nceas/metacat/integration`, or some other clear naming pattern that differentiates the test types clearly.

Reply:
Agreed - my plan here was to at least document what we have now, so it's not quite as confusing, and then do the renames and moving around of tests etc as a separate PR, so as not to blow this one up any further.